### PR TITLE
Avatar styling

### PR DESCRIPTION
**Pull request by libbyeh**
Added avatar images and styling

### CODE REVIEW

File: lib/Player.js [hunk 1]
```diff
@@ -6,14 +6,13 @@ class Player {
...
   increaseScore(score){
     this.score += score;
-    domUpdates.changeScoreboard(this.score);
+    domUpdates.changeScoreboard(score);
   } 
  
   decreaseScore(score) {
     this.score -= score;
-    domUpdates.changeScoreboard(this.score);
+    domUpdates.changeScoreboard(score);
```
Comment (kaohman) [inline lib/Player.js]: Don't want to merge these to master, these are old changes that we updated yesterday I believe.

File: styles.css [hunk 2]
```diff
@@ -164,6 +164,48 @@ h3 {
...
+  margin-left: -122px;
+  width: 9%;
+  z-index: 1;
+}
+
+.avatar1 {
+  background-image: url('assets/alien1.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
```
Comment (kaohman) [inline styles.css]: Suggest adding lines 178-180 to .avatar class, these are used in all three avatars.

File: styles.css [hunk 3]
```diff
@@ -164,6 +164,48 @@ h3 {
...
+
+}
+
+.avatar2 {
+  background-image: url('assets/alien2.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
```
Comment (kaohman) [inline styles.css]: Remove 186-188 after these are moved to .avatar class.

File: styles.css [hunk 4]
```diff
@@ -164,6 +164,48 @@ h3 {
...
+
+}
+
+.avatar3 {
+  background-image: url('assets/alien3.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
```
Comment (kaohman) [inline styles.css]: Remove 194-196 after these are moved to .avatar class.

File: lib/Player.js (hunk 1 above)
Comment (libbyeh) [reply lib/Player.js]: Agree, wont merge this!

**Pull request merged**